### DISCUSSION TITLE
Can't get values from Array

### DISCUSSION THREAD

**Discussion by TomS-** [vuejs/petite-vue#57] [Q&A]
Hey everyone,

I can't seem to get any values from an array fetched from the Shopify Storefront API, I'm getting the message
`petite-vue:1 TypeError: 'get' on proxy: property 'attrs' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '#<Object>' but got '#<Object>')`

This is the API = https://shopify.github.io/js-buy-sdk/

What does this even mean? How do I get around it. For v-for works correctly, but as soon as I try to get data from the array this is what happens.

**Comment by TomS-** [comment]
I switched to AlpineJS V2 and it worked, AlpineJS V3 that is based on Vue's reactive core produces the same error. I think this issue is something bigger than just Petite Vue. 

`(expected '#<Object>' but got '#<Object>')`

This doesn't even make sense to me. 

Found this:
https://github.com/vuejs/vue-next/issues/3024

Which I believe is the same issue, but I don't think PetiteVue has MarkRaw.

**Comment by jokagent** [comment]
Did you try to reset array to plain with usage JSON.parse(JSON.stringify(storefrontArr))?
